### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by Ronserruya**
The original project used appveyor, changed it to use travis + added makefile

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+solution: kin-base-csharp.sln
+
+mono: none
+
+matrix:
+  include:
+    - os: linux
+      dist: xenial
+    - os: osx
+    # - os: windows , very expiremental, does not work yet
```
Comment (yosriz) [inline .travis.yml:11]: Out of curiosity, what's not working in windows? Funny..

**Review by yosriz — APPROVED** [review]
LGTM, Please switch to travis.org instead of .com, as it's free for open source

File: .travis.yml (hunk 1 above)
Comment (Ronserruya) [reply .travis.yml:11]: Windows is still in pre-alpha for travis, when turned it on it tried to run some unix commands (sudo etc), didnt really investigate much

**Comment by Ronserruya** [comment]
> LGTM, Please switch to travis.org instead of .com, as it's free for open source

How do I switch to .org? I've set up everything in .org, I think github just links to .com by default 
https://travis-ci.org/kinecosystem/kin-base-csharp

**Pull request merged**